integration_tests/fixtures/forseti/modules/policy_library/main.tf

```diff
@@ -16,12 +16,17 @@
 
 locals {
   files = [
-    "constraints/cloudsql_location.yaml",
-    "constraints/compute_zone.yaml",
+    "policy-library/lib/constraints.rego",
```

Comment:
Relying on static policies for tests until Forseti has the ability to pull in a bundle.